2996_reflection/reflection.md

```diff
@@ -2027,6 +2065,8 @@ Add a new subsection in [meta]{.sref} after [type.traits]{.sref}:
 
 ```
 namespace std::meta {
+  using info = decltype(^::);
```

Comment:
Surely if the goal is to minimize typing:
```suggestion
  using info = decltype(^0);
```

Reply:
It's not really the goal.  I prefer `decltype(^::)` for emotional reasons.

Reply:
Hehe, okay fair.